### PR TITLE
Fix MSP rx sticks window

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes the MSP emulator radio window, that was broken when moving to modules:

image

It renames the fature too to let it clear the functionality.

### CODE REVIEW

**Comment by McGiverGim** [comment]
Little change to make the margin of the bottom bigger:
image

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
